cmd/init.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"log"
+
+	getter "github.com/hashicorp/go-getter"
+	"github.com/spf13/cobra"
+)
+
+const (
+	furyFile                    = "Furyfile.yml"
+	kustomizationFile           = "kustomization.yaml"
+	httpsDistributionRepoPrefix = "http::https://github.com/sighupio/poc-fury-distribution/releases/download/"
+)
+
+var fileNames = [...]string{furyFile, kustomizationFile}
+var distributionVersion string
+
+func init() {
+	rootCmd.AddCommand(initCmd)
+	initCmd.Flags().StringVar(&distributionVersion, "version", "", "Specify the Kubernetes Fury Distribution version")
+	initCmd.MarkFlagRequired("version")
```

Comment:
can we set a default latest version?

Reply:
Defaults imho are enemies to be avoided generally. Imho is something you must be aware of when you setup a project that inherits many dependencies. We should provide at least a sub command that show all versions and a subcommand --latest for that purpose. But is something that you explicitly set manually

Reply:
Totally agree with @lzecca78.